virtwho/executor.py

```diff
@@ -68,8 +74,41 @@ def _create_virt_backends(self):
                 self.logger.error('Unable to use configuration "%s": %s', name, str(e))
                 continue
             virts.append(virt)
+        self._init_run_status(config_names)
         return virts
 
+    def _init_run_status(self, config_names=[]):
+        # ensure that there is an entry in the status file for each config. We don't care if they work, we
+        # need to record their existence
+        file_lock = FileLock(STATUS_LOCK)
```

Comment:
I would use `with FileLock(STATUS_LOCK) as file_lock` here to be sure that the lock is released, when something unexpected happen. I can see `os.makedirs()` and `json.dump()` between `file_lock.acquire()` and `file_lock.release()`. And you never know. Somebody could add something else in the future.